src/Command/Taxonomy/TaxoDeleteAllCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\Console\Command\Taxonomy;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command as BaseCommand;
```

Comment:
@twhiston  There is no longer need to register this alias. Please update this line 
```
use Symfony\Component\Console\Command\Command as BaseCommand;
```
as this
```
+use Symfony\Component\Console\Command\Command;
```